tests/resources/util.py

```diff
@@ -7,11 +7,13 @@
 TYPES_WITH_CATEGORIES = [DatasetTypes.IMAGE_CLASSIFICATION_MULTICLASS, DatasetTypes.IMAGE_CLASSIFICATION_MULTILABEL, DatasetTypes.IMAGE_OBJECT_DETECTION]
 
 
-def coco_dict_to_manifest(task, coco_dict):
+def coco_dict_to_manifest(task, coco_dict, schema: dict = None):
     if task == DatasetTypes.MULTITASK:
         return coco_dict_to_manifest_multitask(coco_dict[0], coco_dict[1])
-
-    adaptor = CocoManifestAdaptorFactory.create(task)
+    if task == DatasetTypes.KEY_VALUE_PAIR:
```

Comment:
nit: feels like you can just pass task and schema to CocoManifestAdaptorFactory.create(task, schema=schema)

schema would be none, if task != DatasetTypes.KEY_VALUE_PAIR, anyway

Reply:
Because coco manifest adaptors for all other kinds of datasets don't need `schema`, if I make schema as an explicit argument for `.create`, I have to do the if-else check inside `.create()` or change all the coco manifest adaptor interfaces.  Any thoughts?